thunderbird/MozillaThunderbird.spec

```diff
@@ -25,16 +25,14 @@
 # orig_suffix b3
 # major 69
 # mainver %major.99
-%define major          68
-%define mainver        %major.1.0
-%define orig_version   68.1.0
-%define orig_suffix    %{nil}
-%define update_channel release
-%define branding       1
-%define releasedate    20190522150740
-%define source_prefix  thunderbird-%{orig_version}
+%define major           68
+%define mainver         %major.1.1
+%define orig_version    68.1.1
+%define orig_suffix     %{nil}
+%define update_channel  release
+%define releasedate     20190924105435
+%define source_prefix   thunderbird-%{mainver}
 
-# https://bugzilla.suse.com/show_bug.cgi?id=1138688
```

Comment:
referencing restricted bugs in openSUSE packages is bad

Reply:
I would prefer to make that bug public instead. Not sure if thats possible, though...